test/test_nestedtensor.py

```diff
@@ -3385,6 +3385,30 @@ def test_is_contiguous(self, device):
         self.assertTrue(nt_contiguous.is_contiguous(memory_format=torch.contiguous_format))
         self.assertTrue(not nt_noncontiguous.is_contiguous(memory_format=torch.contiguous_format))
         self.assertTrue(nt_contiguous_narrow.is_contiguous(memory_format=torch.contiguous_format))
+    
+    def test_sdpa(self, device):
+        sen1 = torch.randn(11, 1024, dtype=torch.float16, device=device)
+        sen2 = torch.randn(13, 1024, dtype=torch.float16, device=device)
+        x_d = sen1.unsqueeze(0)
+        x_nt = torch.nested.as_nested_tensor([sen1], layout=torch.jagged)
+
+        query = torch.nn.Linear(1024, 1024, bias=False, device=device, dtype=torch.float16)
+        key = torch.nn.Linear(1024, 1024, bias=False, device=device, dtype=torch.float16)
```

Comment:
shouldnt the batch have more than one element?

Reply:
I was trying to get the simplest case working first, I'll add a test with more sentences next